org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterSwitchOnoff.java

```diff
@@ -48,9 +48,9 @@
     private ZclOnOffCluster clusterOnOffServer;
 
     @Override
-    public boolean initializeConverter() {
-        clusterOnOffClient = (ZclOnOffCluster) endpoint.getOutputCluster(ZclOnOffCluster.CLUSTER_ID);
-        clusterOnOffServer = (ZclOnOffCluster) endpoint.getInputCluster(ZclOnOffCluster.CLUSTER_ID);
+    public boolean initializeDevice() {
+        ZclOnOffCluster clusterOnOffClient = (ZclOnOffCluster) endpoint.getOutputCluster(ZclOnOffCluster.CLUSTER_ID);
```

Comment:
My rational is that this method is transitory - ie it shouldn't retain state etc. Ok, I agree that we _could_ use the field instead of the local variable, but to me at least the local variable should be initialised in the ```initializeConverter``` method, and disposed in the ```disposeConverter``` method, and used in the listeners. These methods should not (can not!) rely on the ```initializeDevice``` method initialising this field and I just felt it made it clearer not to use it....

I don't overly mind though if you think it's better to use the field instead for some reason?